reference-architecture/vmware-ansible/docker/vsphere_guest.py

```diff
@@ -0,0 +1,1891 @@
+#!/usr/bin/python
```

Comment:
@detiber 

Here is my PR for ansible code btw - https://github.com/ansible/ansible-modules-core/pull/4266